Example cannot be run with node 4

If I try to run the recommended example with node 4, the following happens:
```
λ npm install
npm WARN package.json basic@1.0.0 No description
npm WARN package.json basic@1.0.0 No repository field.
npm WARN package.json basic@1.0.0 No README data
npm WARN unmet dependency E:\Projects\repos\autodll-webpack-plugin\examples\recommended\node_modules\html-webpack-plugin\node_modules\pretty-error\node_modules\renderkid requires utila@'~0.3' but will load
npm WARN unmet dependency E:\Projects\repos\autodll-webpack-plugin\examples\recommended\node_modules\html-webpack-plugin\node_modules\pretty-error\node_modules\utila,
npm WARN unmet dependency which is version 0.4.0
npm WARN unmet dependency E:\Projects\repos\autodll-webpack-plugin\examples\recommended\node_modules\webpack-dev-server\node_modules\http-proxy-middleware\node_modules\micromatch requires is-glob@'^2.0.1' but will load
npm WARN unmet dependency E:\Projects\repos\autodll-webpack-plugin\examples\recommended\node_modules\webpack-dev-server\node_modules\http-proxy-middleware\node_modules\is-glob,
npm WARN unmet dependency which is version 3.1.0
npm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v4.8.0
npm ERR! npm  v2.15.11
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package webpack@3.0.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer html-webpack-plugin@2.29.0 wants webpack@1 || ^2 || ^2.1.0-beta || ^2.2.0-rc || ^3
npm ERR! peerinvalid Peer webpack-dev-server@2.5.0 wants webpack@^2.2.0
```

I don't think this is a big deal but maybe mention the compatible node version in the readme? You could also include a `engines` key in `package.json` like so:
```js
{
  // ...
  "engines": {
    "node": ">= 6.9.2",
    "npm": ">= 3.10.9"
  },
}
```

--- a/package.json
+++ b/package.json
@@ -44,7 +44,7 @@
         "env",
         {
           "targets": {
-            "node": "current"
+            "node": "4"
           }
         }
       ]

